### ISSUE TITLE
Word 'author is not recognized when View is included

### ISSUE THREAD

**Issue by meijeru** [red/red#2429]
This is a most curious error, occurring when running the result of compiling the following script:
```
Red [Needs: 'View]
author: "Rudolf"
win: layout [a: area 100x100 ""]
win/menu: ["About" about]
win/actors: make object! [on-menu: func [face event][a/text: author]]
view win
```
The error message on clicking `About` is:
```
*** Script Error: author has no value
*** Where: act
*** Stack: view either do-events set do-event-loop awake set do-actor if set do-safe if print = type? try do act
```
When the variable `author` is replaced by one with another name, no error occurs!

**Comment by meijeru** [comment]
Question of leakage??

**Comment by endo64** [comment]
Works normal here. Win10 x64.

**Comment by meijeru** [comment]
I first suspected it was an effect of the IDE used, but the error occurs both with Crimson Editor and with Visual Sturio Code. I use `red-25jan17-080ebed.exe`. It also occurs both with `-t MSDOS` and with `-t Windows`

**Comment by geekyi** [comment]
Can reproduce. Win 10 x64

**Comment by meijeru** [comment]
If @endo64 does not have the error, and @geekyi and I have it, and all of us run W10 x64, what could be the differentiating factor?

**Comment by DideC** [comment]
ON my Win7/64, it compile, but not works as expected : when I click on About, the program leave !

Can you check the .exe properties. The only "author" not in a header is in the [`PE.r` file](https://github.com/red/red/blob/5ecc874df1f8abfea524e0947333090943cef480/system/formats/PE.r#L217)

**Comment by meijeru** [comment]
As noted above, I use `red-25jan17-080ebed.exe`

**Comment by dockimbel** [comment]
The issue is related to View module compilation. The following reduced version exhibits the same bug:
```
Red [Needs: View]

author: "Rudolf"
do [author]
```
while the following variation does not:
```
Red []

author: "Rudolf"
do [author]
```

The internal layout of compilation output has been changed for the needs of libRed, so it is probably the cause of this regression.

**Comment by dockimbel** [comment]
It is a regression caused by the fix for #1829.
